container/ClearSky/src/main/java/io/egen/clearskyboot/constants/HTTPStatus.java

```diff
@@ -0,0 +1,43 @@
+package io.egen.clearskyboot.constants;
+
+public final class HTTPStatus {
```

Comment:
This is for a different purpose actually, this has code and message for @ApiResponse. This is used to denote success and error message types and description on swagger page. I wanted all the description to be consistent. HttpStatus class from SpringFramework has only enum int values but no description for http codes.